.github/workflows/build_push_service.yml

```diff
@@ -6,20 +6,169 @@ name: sc-app-build-push
 
 # Controls when the action will run.
 on:
+  push:
+    branches:
+      - develop
+      - master
+      # release branches are of e.g. release/25.3.0
+      - release/[0-9]+.[0-9]+.[0-9]+
+      # OPS-1664 naming convention <branch>/<JIRA-Ticket ID>-<Jira_Summary>
+      - hotfix/[A-Z]+-[0-9]+-[a-zA-Z0-9]+_[a-zA-Z0-9]+*
+      - feature/[A-Z]+-[0-9]+-[a-zA-Z0-9]+_[a-zA-Z0-9]+*
   workflow_dispatch:
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "deploy"
-  build:
+  # This workflow contains a single job called "build_and_push"
+  build_and_push:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
     env:
       # Setting environment variables here make them available for all steps
-      CI_GITHUB_TRAVISUSER_SWARMVM_KEY: ${{ secrets.CI_GITHUB_TRAVISUSER_SWARMVM_KEY }}
-      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-      DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+      AVSCANFILE_REPO: schulcloud-avcheck-scanfile
+      AVSCANURL_REPO: schulcloud-avcheck-scanurl
+      AVSCANWEBSERVER_REPO: schulcloud-avcheck-scanurl
+      DOCKER_NAMESPACE: schulcloud
+      BRANCH_NAME: ""
+      AVCHECK_VERSION: ""
+      DOCKER_TAG_POSTFIX: ""
+      DOCKER_TAG_SHA: ""
+      JIRA_TicketID: ""
+      BRANCH_PREFIX: ""
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it must be on first place to allow central setting of working directory
       - uses: actions/checkout@v2
+      - name: Get Branch name
+        id: branch
+        uses: nelonoel/branch-name@v1.0.1
+      - name: Set BRANCH_NAME
+        run: echo Set BRANCH_NAME
+        env:
+          BRANCH_NAME: ${{ steps.branch.outputs.BRANCH_NAME }}
+      - name: Read version file
+        # The version to be used for tagging the images is read out of the version file
+        id: read_version
+        uses: juliangruber/read-file-action@v1
+        with:
+          path: ./version
+      - name: Set env AVCHECK_VERSION
+        run: |
+          echo "Version of antivirus_service is ${{ steps.read_version.outputs.content }}"
+          echo "AVCHECK_VERSION=${{ steps.read_version.outputs.content }}" >> $GITHUB_ENV
+      - name: RegEx JIRA ticket ID
+        uses: actions-ecosystem/action-regex-match@v2
+        id: jira
+        with:
+          text: ${{ env.BRANCH_NAME }}
+          regex: '^hotfix|feature\/([A-Z]+-[0-9]+)-[a-zA-Z_]+$'
+          flags: gm
+      - name: Set env var JIRA_TicketID
+        run: |
+          echo "JIRA_TicketID is ${{ steps.jira.outputs.group1 }}"
+          echo JIRA_TicketID=${{ steps.jira.outputs.group1 }} >> $GITHUB_ENV
```

Comment:
Why is it put to ENV vars if is at the output?

Reply:
Other actions have access to env vars, we need this var to build correct dockertag with new naming convention.